amundsen_application/static/js/components/common/ResourceListItem/SchemaInfo/tests/index.spec.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import SanitizedHTML from 'react-sanitized-html';
+
+import { shallow } from 'enzyme';
+
+import { OverlayTrigger, Popover } from 'react-bootstrap';
+
+import SchemaInfo, { SchemaInfoProps } from "components/common/ResourceListItem/SchemaInfo";
+
+describe('SchemaInfo', () => {
+  const setup = (propOverrides?: Partial<SchemaInfoProps>) => {
+    const props: SchemaInfoProps = {
+      schema: 'tableSchema',
+      table: 'tableName',
+      desc: 'schemaDescription',
+      placement: 'bottom',
+      ...propOverrides
+    };
+    const wrapper = shallow(<SchemaInfo {...props} />);
+    return { props, wrapper };
+  };
+
+  describe('render', () => {
+    let props: SchemaInfoProps;
+    let wrapper;
+
+    beforeAll(() => {
+      const setupResult = setup();
+      props = setupResult.props;
+      wrapper = setupResult.wrapper;
+    });
+
+    it('renders OverlayTrigger with correct placement', () => {
+      expect(wrapper.find(OverlayTrigger).props().placement).toEqual(props.placement);
+    });
+
+    it('renders OverlayTrigger with correct popover', () => {
+      const expectedPopover = (
+        <Popover id="popover-trigger-hover-focus">
+          <strong>{ props.schema }:</strong> { props.desc }
+        </Popover>
+      );
+      expect(wrapper.find(OverlayTrigger).props().overlay).toEqual(expectedPopover);
+    });
+
+    it('renders OverlayTrigger with correct schema name', () => {
+      expect(wrapper.find(OverlayTrigger).find('.underline').text()).toEqual(props.schema);
+    });
+
+    it('renders correct table name', () => {
+      expect(wrapper.children().at(1).text()).toEqual('.');
```

Comment:
nit: For this particular text I personally wouldn't go so deep and instead just check that it renders with the correct text: "expect(wrapper.children().text()).toEqual(`${props.schema}.${props.table}`);"

I do see that the test above this ones verifies `props.schema`, but you could consider that that unit test verifies the specific styling that is part of this feature, while this unit test verifies the text for this feature.

Reply:
yes you are right